ways_of_working.md

```diff
@@ -33,6 +33,7 @@ It further outlines expectations and responsibilities with regard to working on
 |  Batool Almarzouq <br>([BatoolMM](https://github.com/BatoolMM)) | Postdoctoral Researcher, KAIMRC and Honorary Research Fellow, The University of Liverpool  | The Turing Way Translation and Localisation Co-Lead | 2020 - Present | - |
 | Alejandro Coca Castro <br>([acocac](https://github.com/acocac)) | Postdoctoral Research Associate, The Alan Turing Institute | The Turing Way Translation and Localisation Co-Lead | 2021 - Present | - |
 | Andrea Sánchez-Tapia <br>([AndreaSanchezTapia](https://github.com/AndreaSanchezTapia)) | Independent researcher | The Turing Way Translation and Localisation Co-Lead | 2021 - Present | - |
+| Melissa Black <br>([melibleq](https://github.com/melibleq)) | Infrastructure and Impact Measurement Manager at Metadocencia | The Turing Way Translation and Localisation Co-Lead | 2023 - Present | - |
```

Comment:
Welcome Melissa (:

Reply:
Thank you so much, @BatoolMM, @acocac and @aleesteele! :)!